packages/eslint-plugin-typescript/src/configs/recommended-package/rules/custom.ts

```diff
@@ -0,0 +1,8 @@
+import { Linter } from 'eslint';
+import { packageForcePrivate } from './custom/package-force-private';
+
+const pluginPrefix = '@cloudflight/typescript';
```

Comment:
This rule is not really typescript specific. I think it should rather be in its own eslint-plugin `@cloudflight/javascript`